packages/runtime/container-runtime/src/summarizerHeuristics.ts

```diff
@@ -104,12 +104,31 @@ export class SummarizeHeuristicRunner implements ISummarizeHeuristicRunner {
         };
     }
 
+    public get idleTime(): number {
+        const maxIdleTime = this.configuration.maxIdleTime;
+        const minIdleTime = this.configuration.minIdleTime;
+        const weightedNumOfOps = getWeightedNumberOfOps(
+            this.heuristicData.numRuntimeOps,
+            this.heuristicData.numNonRuntimeOps,
+            this.configuration.runtimeOpWeight,
+            this.configuration.nonRuntimeOpWeight,
+        );
+        const pToMaxOps = weightedNumOfOps * 1.0 / this.configuration.maxOps;
```

Comment:
Would setting to 0 ever make sense? Why would we summarize when there aren't any new ops?
It would chuck an error here, but maybe we can add an assert saying this number must be > 0.

Reply:
Yes, I don't think it would make sense but what if they set it (by mistake or for some other reason) on the config - then we would have a problem.  I wonder if we should check it even sooner, like when we read the config setting...

Reply:
[AB#758](https://dev.azure.com/fluidframework/internal/_workitems/edit/758)